docs/Comparison.md

```diff
@@ -1,11 +1,93 @@
-# Comparison between the Installed Collector and the OpenTelemetry Collector
+# Comparison between the Installed Collector and OpenTelemetry Collector
+
+## Delivery Phase 1: “Initial Release”
+
+_Expected March 2022_
```

Comment:
Let's remove the phases/dates.  The main reason is this is internal. All other items in the doc are valid.  As we enhance OT and add support for these we can update this.